database/tables.py

```diff
@@ -32,7 +32,7 @@
 
 
 def week_time_range(year: int, week: int,
-                    tz: timezone) -> Tuple[datetime, datetime]:
+                    tz: BaseTzInfo) -> Tuple[datetime, datetime]:
     """Given a year and a week number, returns the time range corresponding.
 
     Note the returned end date matches exactly the start date of the next week
```

Comment:
I cannot comment on L41, so doing it here:

Some year have week 53, 2020 is an example, the first days of January 2021 will be in Week53.
More info [here](https://en.wikipedia.org/wiki/ISO_week_date)